docs/typescript.md

```diff
@@ -1,14 +1,170 @@
-# TypeScript Guidelines
+# **Typescript guidelines**
```

Comment:
```suggestion
# Typescript Guidelines
```
- All words should be capitalized in document title. 
- Applying bold style to a heading doesn't have any effect. All instances of this should be fixed throughout the doc.

Reply:
Gotcha !